libs/angular-accelerator/src/lib/components/search-header/search-header.component.ts

```diff
@@ -1,20 +1,32 @@
 import {
+  AfterContentInit,
   AfterViewInit,
   Component,
   ContentChild,
+  ContentChildren,
   ElementRef,
   EventEmitter,
   Input,
   Output,
+  QueryList,
   TemplateRef,
   ViewChild,
 } from '@angular/core'
 import { Action } from '../page-header/page-header.component'
-import { SearchConfigInfo } from '../../model/search-config-info'
+import { SLOT_SERVICE, SlotService } from '@onecx/angular-remote-components'
+// import { SearchConfigInfo } from '../../model/search-config-info'
+import { FormControlName, FormGroup, FormGroupDirective } from '@angular/forms'
+import { Observable, combineLatest, debounceTime, map, of, startWith } from 'rxjs'
 
 export interface SearchHeaderComponentState {
   activeViewMode?: 'basic' | 'advanced'
-  selectedSearchConfig?: SearchConfigInfo
+  selectedSearchConfig?:
```

Comment:
I would only safe the name or id because it is redundant and possible inconsistent otherwise